metsim/metsim.py

```diff
@@ -275,6 +281,16 @@ def setup_output(self, prototype: xr.Dataset=None):
         dims = ('time', ) + self.domain['mask'].dims
         coords = {'time': times, **self.domain.mask.coords}
         self.output = xr.Dataset(coords=coords)
+        for k, v in self.params.items():
+            # Need to convert some parameters to strings
+            if k in ['start', 'stop', 'annual', 'mtclim_swe_corr']:
+                v = str(v)
+            # Don't include complex types
+            if type(v) is list:
```

Comment:
1) I think it is better to use `isinstance` when checking types.
2) We should check generally for `v` being an iterable. For example, a `tuple` should be treated in the same way here.